ros2launch/ros2launch/command/launch.py

```diff
@@ -44,6 +44,11 @@ def add_arguments(self, parser, cli_name):
         command_group.add_argument(
             '-s', '--show-args', '--show-arguments', default=False, action='store_true',
             help='Show arguments that may be given to the launch file.')
+        parser.add_argument(
+            '-a', '--show-all-subprocesses-output', default=False, action='store_true',
+            help=('Show all launched subprocesses\' output by overriding their output'
+                  ' configuration using the $OVERRIDE_LAUNCH_PROCESS_OUTPUT envvar.')
```

Comment:
Nitpick: remove the `$` since the environment variable name is just `OVERRIDE_LAUNCH_PROCESS_OUTPUT`.

Reply:
Sure. Fixed.